starknet_py/net/account/base_account.py

```diff
@@ -56,6 +61,18 @@ async def get_balance(
         :return: Token balance.
         """
 
+    @abstractmethod
+    async def sign_for_fee_estimate(
+        self, transaction: TypeAccountTransaction
+    ) -> TypeAccountTransaction:
+        """
+        Sign a transaction for a purpose of fee_estimation.
+        Uses transaction version calculated like ``transaction.version + 2 ** 128``.
```

Comment:
Should the exact way of calculating the version be specified here?

Reply:
Good point. Maybe `Should use ...`? And a little explanation of why this is a good idea to estimate_fee with a different signature.